src/LABELS.md

```diff
@@ -8,34 +8,34 @@ Please be sure to review all the [labels per projects](https://github.com/Alaska
 
 The following guide is a description as to the process of managing issues and pull requests within each repo.
 
-It is expected that all who touch an issue or a pull request do their part to ensure that the correct labels are associated to the task or pull request. 
+It is expected that all who touch an issue or a pull request do their part to ensure that the correct labels are associated to the task or pull request.
 
 ### Generic labels
 
 Labels such as `dupicate` or `help wanted` are general indicators that can be appled at any time in the work item's lifecycle.
 
 ### Is there a priority for the ISSUE?
```

Comment:
In keeping with the theme of the other sections (Status and Type) this should be a level 4 sub heading of a level 3 sub heading name "Priority".

Reply:
I see where you are going, but that is not accurate. 

The labels fall into these categories; general, priority. status and type. This may be confusing as this content was not completed.